packages/material-ui-lab/src/SliderStyled/SliderStyled.d.ts

```diff
@@ -48,4 +48,46 @@ export const SliderValueLabel: React.FC<SliderValueLabel>;
  */
 declare const Slider: ExtendSliderUnstyled<SliderStyledTypeMap>;
 
+export type SliderStyledClassKey =
+  /** Styles applied to the root element. */
```

Comment:
Should it be

```suggestion
  /** Class name applied to the root element. */
```

and so on?

Reply:
Yeah I think this make more sense now..

Reply:
Updated